wordpress-mobile/WordPress-Android
# Fix/14993 multiple sites blogging reminders opening issues

Fixes #14993 

<img width=320 src="https://user-images.githubusercontent.com/990349/125239852-9c94dd80-e32c-11eb-946c-8284c6d1a2c2.png"/>

To test:

- Turn on the blogging reminders flag (Me/App settings/Test feature configuration - BloggingRemindersFeatureConfig)
- Notice the blogging goals option appears (My Site > Settings > Blogging reminders)
- Click on for e.g. "Twice a week" button under Blogging goals setting
- Notice the blogging reminders prologue screen is shown
- Click on "Set goals"
- Select a day for blogging when screen is shown, then ensure "Update" button is enabled
- Click on "Update" button
- Notice the epilogue screen is shown. Done button is enabled, Clicking Done button should dismiss bottomsheet
- Repeat the steps for other sites that you may have
- Wait for the next day, verify that notifications appear for all the sites that have been setup for reminders



## Regression Notes
1. Potential unintended areas of impact
Site Settings

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Blogging reminders flow from Site Settings

3. What automated tests I added (or what prevented me from doing so)
Unit tests

PR submission checklist:
